project-base/config/paths.yaml

```diff
@@ -7,7 +7,7 @@ parameters:
     shopsys.domain_images_dir: '/web%shopsys.domain_images_url_prefix%'
     shopsys.domain_images_url_prefix: '/%shopsys.content_dir_name%/admin/images/domain'
     shopsys.domain_urls_config_filepath: '%shopsys.root_dir%/config/domains_urls.yaml'
-    shopsys.error_pages_dir: '%shopsys.root_dir%/var/errorPages/'
+    shopsys.error_pages_dir: '/var/errorPages/'
```

Comment:
Is necessary to update this path? Isn't it broken in monorepo?

Reply:
Flysystem is configured to project root by default. So I need to change that because I don't want to have errorPages in path `/var/www/html/var/errorPages/` on S3 file storage. I do not know if it will break monorepo.

Reply:
It may not work without Flysystem on localhost, but I didn't test it.

Reply:
It works :)

Reply:
@pk16011990 It's working like uploading images etc. So if there are working upload images on localhost this should also work.